core/DataAccess/ArchivingDbAdapter.php

```diff
@@ -98,4 +111,25 @@ private function logSql($sql)
             $this->logger->debug($sql);
         }
     }
+
+    private function callFunction($function) {
+
+        $args = func_get_args();
+        unset($args[0]);
+
+        try {
+            return call_user_func_array([$this->wrapped, $function], $args);
+        } catch (\Exception $e) {
+            if ($this->isErrNo($e, \Piwik\Updater\Migration\Db::ERROR_CODE_MAX_EXECUTION_TIME_EXCEEDED_QUERY_INTERRUPTED) ||
+                $this->isErrNo($e, \Piwik\Updater\Migration\Db::ERROR_CODE_MAX_EXECUTION_TIME_EXCEEDED_SORT_ABORTED)
+            )
+            {
+                $this->logger->info('Archiver query exceeded maximum execution time: {details}',
```

Comment:
I doubt that many people would actually log a `info` on an instance. As the exception is rethrown, and the archiving will fail with an error in any case, wouldn't it be more helpful to log that as a warning, so it appears with normal log configuration?

Reply:
Yes, makes sense :+1: